cjs/html/style-element.js

```diff
@@ -1,16 +1,51 @@
 'use strict';
+const {parse} = require('cssom');
+
 const {registerHTMLClass} = require('../shared/register-html-class.js');
 
 const {TextElement} = require('./text-element.js');
 
 const tagName = 'style';
 
+const SHEET = Symbol('sheet');
```

Comment:
do you mind moving this into https://github.com/WebReflection/linkedom/blob/main/esm/shared/symbols.js ?

Reply:
P.S. apologies I've commented on the `cjs` file, I meant it on the `esm` one.

Reply:
I put it there because I saw the same structure in "template-element" with CONTENT symbol.
Do you want me to move that Symbol too? And maybe do a quick check for eventual others not shared symbols?

Reply:
Nope, thanks though, that’s been me lazy then, but moving those would be unrelated to this MR, I’ll take care of that

Reply:
I've added an issue for that https://github.com/WebReflection/linkedom/issues/77